doc/examples.xml

```diff
@@ -309,3 +309,35 @@ gap> GeneralisedPetersenGraph(IsMutableDigraph, 9, 4);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="LollipopGraph">
+<ManSection>
+  <Oper Name="LollipopGraph" Arg="[filt, ]n, k"/>
+  <Returns>A digraph</Returns>
+  <Description>
+    If <A>m</A> and <A>n</A> are positive integers and <A>k</A>, 
```

Comment:
What's `m`? I can't see it in the list of arguments

Reply:
Also, `k` is not explained. Looks like names mismatch with the actual code.

Reply:
@alex-konovalov thanks for this! Apologies for the confusion! :)